src/reasonFirebase.re

```diff
@@ -109,7 +107,7 @@ module Storage = {
     [@bs.send] external path : (t, ~path: string) => t = "";
     [@bs.send]
     external put :
-      (t, ~data: Window.File.t, ~metadata: Js.t('a)=?, unit) => Js.Promise.t(UploadTask.t) =
+      (t, ~data: Js.Array.t(int), ~metadata: Js.t('a)=?, unit) => Js.Promise.t(UploadTask.t) =
```

Comment:
Maybe just have it `'a` since Js.Array of int is definitely not correct in the browser.